### DISCUSSION TITLE
Broader HTML tag coverage

### DISCUSSION THREAD

**Discussion by MarkusJohansen** [ChimeraCSS/ChimeraCSS#72] [Ideas]
Ideally ChimeraCSS would cover every element tag, to provide the highest possible level of consistency for the user. This is not the case as per today. We will welcome element styling for uncovered elements.

**Describe the solution you'd like**
The disussion can be closed when these are implemented:

```HTML
<area>
<audio>
<b>
<br> 
<caption>
<cite>
<col>
<colgroup>
<dd>
<del>
<details>
<dfn>
<dialog>
<dl>
<dt>
<em>
<fieldset>
<legend>
<map>
<menu>
<meter>
<optgroup>
<option>
<output>
<param>
<progress>
<q>
<s>
<samp>
<select>
<small>
<strong>
<sub>
<summary>
<sup>
<tfoot>
<thead>
<time>
<u>
<details>
<summary>
```

**Additional context**
If relevant uncovered tags is discovered, and not included in the list, please open an issue or comment! If one of the tags should be removed from the list, please do the same!

Only HTML5 supported tags!

**Comment by diomed** [comment]
details and summary would really be a nice add
